### DISCUSSION TITLE
Is there a way to save models into desired location?

### DISCUSSION THREAD

**Discussion by erdkse** [mindee/doctr#1122] [Q&A] [answered]
I am using docTR in the Kubernetes environment and every time a new pod is up the models are downloaded again and again so I wonder If I can cache them by providing a custom location.

**Comment by felixdittrich92** [comment, marked as answer]
Hi @erdkse :wave: ,
sry for the late reply you can change the cache dir as described in -> https://mindee.github.io/doctr/using_doctr/running_on_aws.html

**Comment by erdkse** [comment]
Thank you so much! I tried it now and it is working as expected.
